packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -148,6 +156,141 @@ export class LitNodeClientNodeJs extends LitCore {
     return claimRes;
   };
 
+  // ========== Rate Limit NFT ==========
+
+  // TODO: Add support for browser feature/lit-2321-js-sdk-add-browser-support-for-createrlidelegationauthsig
+  createRliDelegationAuthSig = async ({
+    dAppOwnerWallet,
+    rliTokenId,
+    addresses,
+    uses,
+    domain,
+    expiration,
+  }: {
+    dAppOwnerWallet: ethers.Wallet;
```

Comment:
We should follow up with a change to allow other types of signing interaction. `Providers` can also support `eth_personalSign` requests